user/example-models/fuzzy_model.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+
+from h1st.model.fuzzy import (
+    FuzzyLogicModeler, 
+    FuzzyLogicModel, 
+    FuzzyLogicRules,
+    FuzzyMembership as fm
+)
+
+
+def build_fuzzy_model():
+    flr = FuzzyLogicRules()
+    flr.add_variable(
```

Comment:
Consider restructuring into two collections, `fuzzy_rules` and `fuzzy_variables`. Add rules to `fuzzy_rules`, and variables to `fuzzy_variables`. To link them together into a single reference, have a `fuzzy_model` that contains `fuzzy_rules` and `fuzzy_variables`.

It is not good design to place `fuzzy_variables` within `fuzzy_rules`.